scope

```diff
@@ -182,6 +182,7 @@ launch_command() {
 }
 
 launch_docker4mac_app_command() {
+    dry_run
```

Comment:
You don't really want to invoke a side-effecting command here, seeing as this function is simply meant to return a command (as the name suggests).

So I'd put this just before the `check_not_running` calls, which is the same place as in the non-d4m case.